api.planx.uk/server.ts

```diff
@@ -248,11 +250,7 @@ app.get("/hasura", async function (_req, res, next) {
  *                  type: string
  *                  example: 2023-08-11T11:28:38.237493+00:00
  */
-app.get("/me", useJWT, async function (req, res, next) {
-  // useJWT will return 401 if the JWT is missing or malformed
-  if (!req.user?.sub)
-    next({ status: 401, message: "User ID missing from JWT" });
-
+app.get("/me", usePlatformAdminAuth, async function (req, res, next) {
```

Comment:
will any signed in user (including non-platform admins) still be able to call `/me` if we're using this auth method here? if not, do we want that to be possible? thinking forward to the `/me` endpoint perhaps underpinning a frontend user store & ability to display your use rdetails in the editor?

Reply:
Yeah this is a good catch - I added `usePlatformAdminAuth` as I was viewing this as a nice way for devs to check their status.

I'll make it accessible to any logged on user again on a follow up PR 👍 

For a user store, we could populate it a number of ways - 
- Calling `/me`
- Calling `$admin.user.getByEmail()` directly
- Enriching the JWT with all required user details

I think you're right here - `/me` may be the simplest method